### ISSUE TITLE
Option to use space as thousands seperator

### ISSUE THREAD

**Issue by dartiss** [dartiss/open-currency-converter#28]
Requested here - https://wordpress.org/support/topic/space-instead-of-comma-for-the-separation-of-thousands/

**Comment by Presskopp** [comment]
This would be a lifesaver for me +1

**Comment by Deralict** [comment]
This would save a tonne of headaches, +1!!!!!

**Comment by SimonVetterli** [comment]
Could be any character, such as " ' " instead of a comma.

**Comment by dartiss** [comment]
Indeed. My idea would be simply to allow you to provide your own seperating character.

**Comment by SimonVetterli** [comment]
This sounds great. Would be a real advantage. +1

**Comment by dartiss** [comment]
Also allow the separator to be suppressed... https://wordpress.org/support/topic/remove-thousand-indicator/

**Comment by MVColombie** [comment]
Hello @dartiss, any news about this feature ?

**Comment by dartiss** [comment]
No plans yet, but this one is a priority - it's the next thing I'll get implemented.
